src/utils/PropertiesUtils.tsx

```diff
@@ -0,0 +1,5 @@
+import properties from '../properties.json';
```

Comment:
by importing the `.json` file it gets bundled into `/static/js/main.f9371155.chunk.js` this was the case previously already.
Please allow the object (JSON) to be in `index.html` this way an extra roundtrip to the backend is prevented.